mobile/src/splash-screen/splash-screen.tsx

```diff
@@ -166,30 +173,12 @@ class SplashScreenImpl extends React.Component<SplashScreenStateProps & SplashSc
 
     //----------------------------------------------------------------------------
     private renderChoosePinCode(): JSX.Element {
-        return (
-            <ArcadiaPinCode
-                status={'choose'}
-                storePin={(pin: string) => {
-                    this.pinStore(pin);
-                    this.onSuccess();
-                }
-                }
-                useLogoutButton={true}
-                onClickLogoutButton={this.onLockedScreenClose}/>
-        );
+        return this.renderPinCode('choose');
     }
 
     //----------------------------------------------------------------------------
     private renderAskPinCode(storedPin: string): JSX.Element {
-        return (
-            <ArcadiaPinCode
-                status={'enter'}
-                storedPin={storedPin}
-                finishProcess={this.onSuccess}
-                useLogoutButton={true}
-                onClickLogoutButton={this.onLockedScreenClose}
-            />
-        );
+        return this.renderPinCode('enter', storedPin);
```

Comment:
Do we really need this one line function (and renderChoosePinCode)? renderPinCode can be called directly with an appropriate params

Reply:
Done. ad277a705312a74915eb47beccd72911ac185116